### PR TITLE
added categories

### PR DESCRIPTION
**Pull request by yos244**

### CODE REVIEW

File: src/components/CatFilters.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+export const CatFilters = ({ revList, innerUrl }) => {
+  const [filteredRev, setFilteredRev] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [activeLink, setActiveLink] = useState(false);
+
+  useEffect(() => {
+    setFilteredRev((currRev) => {
+      setLoading(true);
+      const newList = revList.filter((rev) => {
```
Comment (JVal-NC) [inline src/components/CatFilters.jsx:12]: This needs to be done using request queries, have a look at how your backend can filter getReviews based on this.

File: src/App.js [hunk 2]
```diff
@@ -7,35 +7,56 @@ import { HomeButton } from "./components/HomeButton";
...
+import { CatFilters } from "./components/CatFilters";
+import { Categories } from "./components/Categories";
 
 function App() {
+  const [revList, setRevList] = useState([]);
   const [loading, setLoading] = useState(true);
   const [reviewId, setReviewId] = useState("");
-  const [username, setUsername] = useState("grumpy19")
+  const [username, setUsername] = useState("grumpy19");
+  const [innerUrl, setInnerUrl] = useState("");
```
Comment (JVal-NC) [inline src/App.js:18]: Have a look at using react router parametric endpoints and useParams to make the way state is handled cleaner.

**Review by JVal-NC — COMMENTED** [review]
See review comments, please amend and submit new PR.

File: src/components/CatFilters.jsx (hunk 1 above)
Comment (yos244) [reply src/components/CatFilters.jsx:14]: my backend does not filter based on category

**Pull request merged**